passport/index.js

```diff
@@ -17,15 +16,15 @@ passport.deserializeUser((id, done) => {
 		{ _id: id },
 		'username',
 		(err, user) => {
-			console.log('*** Deserialize user, user:')
-			console.log(user)
-			console.log('--------------')
-			done(null, user)
+			console.log('*** Deserialize user, user:');
```

Comment:
This deserialize only returns the username and id to the front end. I will have to figure out how to return the votedProjects along with it so we can use it on the front end to check which projects have been voted for the user.